p/api/fever.php

```diff
@@ -288,7 +288,7 @@ public function wrap($status, array $reply = array())
 		$arr = array('api_version' => self::API_LEVEL, 'auth' => $status);
 
 		if ($status === self::STATUS_OK) {
-			$arr['last_refreshed_on_time'] = (string) $this->lastRefreshedOnTime();
```

Comment:
@kevinpapst It was [a long time ago](https://github.com/FreshRSS/FreshRSS/pull/1836), but do you happen to remember why this was casted to a string?
https://github.com/FreshRSS/FreshRSS/issues/2940

Reply:
Hahaha, can you remember why you did things last months or even two years ago? 😁 

Short answer: no, unfortunately I can't 😞 looks correct when checking the API definition.
I would question if there was a bug at any point or if that change is only happening to be compatible with the "official documentation". I never had troubles with any clients I tested.

Offtopic: I'd really like to help more, but I don't have the time to take care of any other projects that my ones currently... sorry that you have to take care of the API now! I know its a burden (but still great for the iOS users).

Reply:
No worries, and sorry for the quick answer :-)
And if you are still using it, could you try that it still works with your favourite app after this fix? I can merge in the master branch if that can help.
Also, I am making a table overview of clients, and feedback welcome (I have nothing iOS to try myself) https://github.com/FreshRSS/FreshRSS/pull/2942